package.json

```diff
@@ -38,11 +38,11 @@
   },
   "github": "https://github.com/marionettejs/backbone.babysitter",
   "peerDependencies": {
-    "backbone": ">=0.9.9 <=1.3.2",
-    "underscore": ">=1.4.0 <=1.8.3"
+    "backbone": "^1.3.3",
+    "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "backbone": ">=0.9.9 <=1.3.2",
+    "backbone": ">=1.3.3 <1.4.0",
```

Comment:
I think I can switch this to `>=0.9.9` since it really only uses `Backbone` as a namespace object.